.gitignore

```diff
@@ -0,0 +1,4 @@
+./bin/
```

Comment:
`/bin/` would be better (to make it clearer that it's relative to the root of the repo), but this works too.